en/docs/install-and-setup/install/adding-observability-for-icp.md

```
@@ -0,0 +1,535 @@
+
+```toml
+opensearchUrl = "http://localhost:9200"
+```
+
+!!! warning
+    The ICP config file ships with these keys commented out near the bottom of the file, after `[ballerina.http.traceLogAdvancedConfig]`. **Do not uncomment those lines.** Because they fall under a `[section]` header, Ballerina treats them as section-scoped values and rejects them. Always add the OpenSearch keys before the first `[section]` header.
+
+
+Restart ICP Server after saving. Look for this log line to confirm:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)